server/src/routes/ethereum/operational.js

```diff
@@ -14,50 +20,50 @@ router.get('/', (_req, res) => {
   });
 });
 
+/**
+ * @swagger
+ * /api/v1/ethereum/operational/storage-constraint:
+ *  get:
+ *      description: Returns plot points for the storage constraint factor in the operational layer for Ethereum - TEST EXECUTION MAY BE SLOW
+ *      responses:
+ *          200:
+ *              description: Successful response
+ */
+
 router.get('/storage-constraint', async (_req, res) => {
   const resp = await fetch(
-    'https://etherscan.io/chartsync/chaindefault?output=csv',
+    'https://api.blockchair.com/ethereum/blocks?a=date,sum(size)',
     {
       method: 'GET',
     }
   );
-  const respText = await resp.text();
-  const respCsv = await neatCsv(respText);
-  const mid = Math.floor(respCsv.length / 2);
-  const gethBlockchainSizes = respCsv.slice(0, mid);
-  const openEthereumBlockchainSizes = respCsv.slice(mid + 1, respCsv.length);
 
+  const respText = await resp.json();
+  const points = respText.data;
   const compareInterval = 14;
-  const calculatePlotPoints = (
-    { UnixTimeStamp: unixTimeStamp, Value: value },
-    index,
-    array
-  ) => {
+  // eslint-disable-next-line no-empty-pattern
+  const calculatePlotPoints = ({}, index, array) => {
     if (index < compareInterval) {
       return {};
     }
-    const x = new Date(unixTimeStamp * 1000).toISOString().split('T')[0];
-    const y = value / array[index - compareInterval].Value;
-    return { x, y };
+    return {
+      x: array[index].date,
+      y:
+        parseInt(array[index]['sum(size)'], 10) /
+        parseInt(array[index - compareInterval]['sum(size)'], 10),
+    };
```

Comment:
```suggestion
    return {
      x: date,
      y: blockChainSize / array[index - compareInterval]['sum(size)'],
    };
```
No need for `parseInt()` as it's already in number form. After object de-structuring you can use de-structured variables above